app/models/account.rb

```diff
@@ -95,6 +95,22 @@ def reset!
     RedisEndpoint.reset!
   end
 
+  # Get admin emails associated with this account/site
+  def admin_emails
```

Comment:
Can you add coverage for the changes to this file to `spec/models/account_spec.rb`?